### PR TITLE
Gpv 743 remove root dependency

### PR DESCRIPTION
**Pull request by avenkatram85**
- Transferred user dependency from `root` to `gpadmin`
- User can login to `gpadmin` user and run TPC-DS from any location 
- Updated `README.md` with latest changes
- Removed INSTALL_DIR
- Removed creation or modification of tpcds_variables.sh by tpcds.sh

Co-authored-by: Deena Venkatesan vdeenathayal@vmware.com
Co-authored-by: Venkatram Amalanathan avenkatram@vmware.com
Co-authored-by: Stu Pollock <stuartp@vmware.com>
Co-authored-by: Mike Nemesh <nemeshm@vmware.com>

### CODE REVIEW

**Review by sumitbasumallick — APPROVED** [review]
+1

File: README.md [hunk 1]
```diff
@@ -63,12 +52,24 @@ yum -y install build-essential
...
+ssh gpadmin@mdw
+curl -LO https://github.com/pivotal/TPC-DS/archive/refs/tags/v3.0.0.tar.gz
+tar xzf v3.0.0.tar.gz
+mv TPC-DS-3.0.0 TPC-DS
+```
+
 ## Usage
 
-To run the benchmark, just login as `root` on `mdw:
+To run the benchmark, just login as `gpadmin` on `mdw:
```
Comment (stuart-pollock) [inline README.md:68]:
```suggestion
To run the benchmark, login as `gpadmin` on `mdw`:
```

File: tpcds.sh [hunk 2]
```diff
@@ -1,151 +1,76 @@
...
-	echo ""
-	local WHOAMI=`whoami`
-	if [ "$WHOAMI" != "root" ]; then
-		echo "Script must be executed as root!"
-		exit 1
-	fi
+check_user() {
+  echo "############################################################################"
+  echo "Ensure gpadmin is executing this script."
+  echo "############################################################################"
```
Comment (stuart-pollock) [inline tpcds.sh:60]: We should use ${ADMIN_USER} instead of `gpadmin`

Comment (mnemesh-pivotal) [reply tpcds.sh:60]: Agreed

**Pull request merged**